podium-gateway/src/main/webapp/app/backoffice/modules/organisation/organisation.model.ts

```diff
@@ -17,13 +17,25 @@ export class Organisation {
     uuid?: string;
     name?: string;
     shortName?: string;
-    deleted?: boolean;
     activated?: boolean;
     organisationUuid?: string;
     public roles?: Role[];
     public requestTypes?: RequestType[];
 
-    constructor() {
```

Comment:
Its not very pretty to use constructor based instantiation. I would prefer an Object.assign() approach
such as: 

`let organisation: Organisation = Object.assign(new Organisation(), JSON.parse(jsonResponse));`